src/main/java/com/agilethought/atsceapi/service/implementation/UserServiceImpl.java

```diff
@@ -118,11 +118,20 @@ public UpdateUserResponse updateUser(UpdateUserRequest request, String id) {
 			throw new NotFoundException(
 					String.format(NOT_FOUND_RESOURCE, USER, id)
 			);
+		User userToBeUpdated = userFoundById.get();
 		request.setId(id);
-		User user = orikaMapperFacade.map(request, User.class);
-		userValidator.validate(user);
-		setLetterCases(user);
-		User updatedUser = userRepository.save(user);
+		User userUpdatedFields = orikaMapperFacade.map(request, User.class);
```

Comment:
Previously... @graciela24 added this as a validation. I wpuld like to follow same way. Pleae move this as part of email 
validations.

Reply:
I'm sorry, I don't understand which is the part I have to move to the email validations. The marked lines I see in your request are to invoke the validations, set the letter cases, and save the user in the database, and I see those operations are out of the validations in another operations.